docs/ADDRESS_LOOKUP.md

```diff
@@ -11,7 +11,63 @@
 - If you're integrating with Drop-in:
     - Implement the mandatory `onAddressLookupQueryChanged(query: String)` callback and optional `onAddressLookupCompletion(lookupAddress: LookupAddress)` callback.
     - Pass the result of these actions by using `AddressLookupDropInServiceResult` class.
+      ```kotlin
+      override fun onAddressLookupQueryChanged(query: String) {
+          // Use the query parameter to make a call to your desired maps endpoint 
+          // Map the address objects LookupAddress object and pass it back to Drop-in as follows:
+          val options: List<LookupAddress> = // mapped list of address objects
+          sendAddressLookupResult(AddressLookupDropInServiceResult.LookupResult(options))  
+      }
+
+      override fun onAddressLookupCompletion(lookupAddress: LookupAddress): Boolean {
+          // Optionally use this callback to get the complete details for a LookupAddress option selected by the shopper
+          // If you need to use this callback to make a call to get complete details for the selected address option you must return true.
+          // You can also use this function conditionally. In that case, return true if you perform an api call, false otherwise.
+          // Make a call to your desired maps endpoint and pass the complete LookupAddress object back to Drop-in as follows:
+          val shouldCompleteAddress = // whether an api call needs to be made or not
+          if (shouldCompleteAddress) {
+              val completeLookupAddress = // complete LookupAddress object
+              sendAddressLookupResult(AddressLookupDropInServiceResult.LookupComplete(completeLookupAddress))
+              return true
+          }
+          // If you do not need to use this callback either do not implement it, or return false.
+          return false
+      } 
+      ```
 - If you're integrating with standalone `CardComponent`:
     - Set `AddressLookupCallback` via `CardComponent.setAddressLookupCallback(AddressLookupCallback)` function to receive the related events.
     - Pass the result of these actions by calling `CardComponent.setAddressLookupResult(addressLookupResult: AddressLookupResult)`.
+    ```kotlin
+    cardComponent.setAddressLookupCallback(object : AddressLookupCallback {
+        override fun onQueryChanged(query: String) {
+            // Use the query parameter to make a call to your desired maps endpoint
+            // Map the address objects LookupAddress object and pass it back to Drop-in as follows:
```

Comment:
`Drop-in` should be fixed

Reply:
what needs to be fixed there? we always use it like this.
<img width="331" alt="Screenshot 2024-03-18 at 13 25 07" src="https://github.com/Adyen/adyen-android/assets/6615094/22a62670-076e-4437-84c8-b359c9158cbe">
<img width="862" alt="Screenshot 2024-03-18 at 13 25 22" src="https://github.com/Adyen/adyen-android/assets/6615094/beb2e7a6-be56-4d64-867f-50fb36a08780">
<img width="859" alt="Screenshot 2024-03-18 at 13 25 36" src="https://github.com/Adyen/adyen-android/assets/6615094/a194d27d-29fa-4c07-b3fe-c53586d8ef60">